docs/reference/migration/migrate_7_7.asciidoc

```diff
@@ -16,7 +16,21 @@ coming[7.7.0]
 
 //tag::notable-breaking-changes[]
 
-//end::notable-breaking-changes[]
+[discrete]
+[[breaking_77_ml_changes]]
```

Comment:
I'm wondering if this change is that much breaking. In the version I tested before implementing my text-disallowing PR the dependent variable of type `text` did not work anyway. It's just the error message was different...
@dimitris-athanasiou: WDYT?

Reply:
Ah, so if it never worked I can cancel this breaking change and remove the text data types from previous versions too.  Do you agree @sophiec20 ?

Reply:
agreed - not needed to be listed as a breaking change - because it has never worked, it is still experimental, and it would add noise to breaking changes